Sources/Vapor/Utilities/FileIO.swift

```diff
@@ -407,6 +411,8 @@ public struct FileIO: Sendable {
     ///        print("chunk: \(data)")
     ///    }
     ///
+    ///    **Warning** it's the caller's responsibility to close the file handle provided in ``FileChunks`` when finished.
+    ///
```

Comment:
Use a warning callout instead:
```suggestion
    ///  > Warning: It is the caller's responsibility to close the file handle provided in ``FileChunks`` when finished.
    ///
```